packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -130,6 +131,7 @@ export const Checkbox: FC<CheckboxProps> = ({
         <Pressable
```

Comment:
Does it cause issues with this being on the Pressable? Namely, do the a11yLabel overrides still work correctly when they're now nested a layer lower than the a11yValue? The PR video only highlights options 3/4 which don't have a label override.

Reply:
Confirmed that the a11y label overrides still work correctly with voiceover